src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/reports/ZipFilesSummaryRepository.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity.reports;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+public interface ZipFilesSummaryRepository extends JpaRepository<Envelope, UUID> {
+
+    @Query(
+        nativeQuery = true,
+        value = "SELECT env.jurisdiction, env.status, e1.container, "
```

Comment:
In case of an invalid files there is no envelope in db and status will be null.
Is that what we want?

Perhaps we could use the last event instead?

Reply:
check with Ade. he is happy to have empty status for invalid zipfiles.

Reply:
if the only event is `ZIPFILE_PROCESSING_STARTED` it will also result in null status, while we don't know yet if the file is invalid

Reply:
yes! Ade wants both envelope status and latest event of the zipfile. When I asked him to choose one of them, he chose envelope status. 
The query would be expensive to add another join to get the latest event of the zipfile from `process_events`

Reply:
latest event is effectively envelope status

Reply:
@rkondratowicz I've updated the query to get the latest event of the zipfile instead of envelope status.